doc/release/RELEASE-NOTES.md

```
@@ -10,8 +10,31 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * Product name: Open Supply Hub
 * Release date: November 16, 2024
 
+### Database changes
+#### Migrations:
```

Markdownlint

Punctuation: ':'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)